shared/design/src/components/question/README.md

```
@@ -279,14 +293,15 @@ Figma의 Question 컴포넌트 세트는 다음과 같이 구성되어 있었습
 ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)